### PR TITLE
[Bash/it] Italian translation of bash guide

### PR DESCRIPTION
**Pull request by sinkswim**
Italian translation of bash guide

### CODE REVIEW

**Comment by vendethiel** [comment]
:+1:

File: it-it/bash-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+# il suo nome senza $. Se vuoi usare il valore della variabile, devi usare $.
+# Nota che ' (singolo apice) non espande le variabili!
+
+# Sostituzione di stringhe nelle variabili
+echo ${VARIABILE/Una/A}
+# Questo sostituirà la prima occorrenza di "Una" con "La"
+
+# Sottostringa di una variabile
+echo ${VARIABILE:0:7}
+# Questo ritornera solamente i primi 7 caratteri
```
Comment (geoffliu) [inline it-it/bash-it.html.markdown]: ritornerà

**Comment by sinkswim** [comment]
Ops. Fixed. Thanks.

**Comment by geoffliu** [comment]
Nice, thanks!

**Pull request merged**